test/integration/singlecluster/controller/dra/suite_test.go

```diff
@@ -82,59 +82,64 @@ var _ = ginkgo.AfterSuite(func() {
 })
 
 // Manager setup used by tests to start controllers with DRA ConfigMap configuration
-func managerSetup(ctx context.Context, mgr manager.Manager) {
-	// Indexes
-	err := indexer.Setup(ctx, mgr.GetFieldIndexer())
-	gomega.Expect(err).NotTo(gomega.HaveOccurred())
-
-	// Webhooks
-	failedWebhook, err := webhooks.Setup(mgr, config.MultiKueueDispatcherModeAllAtOnce)
-	gomega.Expect(err).ToNot(gomega.HaveOccurred(), "webhook", failedWebhook)
-
-	mappings := []config.DeviceClassMapping{
-		{
-			Name:             corev1.ResourceName("foo"),
-			DeviceClassNames: []corev1.ResourceName{"foo.example.com"},
-		},
-		{
-			Name:             corev1.ResourceName("res-1"),
-			DeviceClassNames: []corev1.ResourceName{"test-deviceclass-1"},
-		},
-		{
-			Name:             corev1.ResourceName("res-2"),
-			DeviceClassNames: []corev1.ResourceName{"test-deviceclass-2"},
-		},
-	}
+func managerSetup(modifyConfig func(*config.Configuration)) framework.ManagerSetup {
```

Comment:
Changes here are almost entirely from indentation, so the "hide whitepsace" checkbox should show a more meaningful diff.